frontend/packages/console-plugin-sdk/src/typings/providers.ts

```diff
@@ -0,0 +1,20 @@
+import { ProviderExoticComponent } from 'react';
+import { Extension } from './base';
+
+namespace ExtensionProperties {
+  export interface ContextProvider<T> {
+    /** Context Provider Exotic Component. */
+    Provider: ProviderExoticComponent<T>;
```

Comment:
I think this is the correct type:
```suggestion
    Provider: React.Provider<T>;
```